src/PAModel/Entropy.cs

```diff
@@ -58,6 +58,9 @@ internal class PropertyEntropy
         // Key is resource name, value is filename
         public Dictionary<string, string> LocalResourceFileNames { get; set; } = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
 
+        // Some of the properties have different value of InvariantScript than the DefaultScript which should be the case, hence need special handling.
```

Comment:
maybe "Some Component Function Parameter Properties are serialized with a different InvariantScript and DefaultScript. We show the InvariantScript in the .yaml, and persist the difference in Entropy to ensure we have accurate roundtripping behavior"

Reply:
Also, call out what the key and value are in the comment.